cache/repo_cache.go

```diff
@@ -66,6 +69,10 @@ type RepoCache struct {
 
 	// the user identity's id, if known
 	userIdentityId entity.Id
+
+	// the cache of commits
+	muCommit sync.RWMutex
+	commits  map[repository.Hash]*object.Commit
```

Comment:
Is that really useful? Maybe I'm missing something but it looks to me that once a commit is read for validation, it's not useful anymore as it won't be shared with any other identities.

Reply:
Yes, it does not seem useful.